docker/pusher.go

```diff
@@ -40,6 +40,7 @@ type PusherDockerClient interface {
 	ImageLoad(context.Context, io.Reader, bool) (types.ImageLoadResponse, error)
 	ImageTag(context.Context, string, string) error
 	ImagePush(ctx context.Context, ref string, options types.ImagePushOptions) (io.ReadCloser, error)
+	ImageInspectWithRaw(context.Context, string) (types.ImageInspect, []byte, error)
```

Comment:
Sorry for my ignorance. Is this just needed because of the bump in the docker dependency version?

Reply:
Probably dep to go-containers lib.

Reply:
Yep, that's an updated interface from go-containers. Should I refactor it to Matej's suggestion? It's definitely cleaner.